src/Gateways/AbstractGateway.php

```diff
@@ -1103,6 +1103,15 @@ public function get_decline_message( string $response_code ) {
 	public static function addCaptureOrderAction( $actions ) {
 		global $theorder;
 
+		if (
+			$theorder->get_payment_method() == 'globalpayments_gpapi' &&
+			$theorder->get_status() == 'pending' &&
+			! empty( $theorder->get_transaction_id() )
+		) {
+			$actions['capture_credit_card_authorization'] = 'Capture credit card authorization';
```

Comment:
Let's have the text inside a variable, because we have it two times in this function.